src/hotspot/share/runtime/thread.cpp

```diff
@@ -902,100 +902,23 @@ char java_runtime_version[128] = "";
 char java_runtime_vendor_version[128] = "";
 char java_runtime_vendor_vm_bug_url[128] = "";
```

Comment:
Should these be static?

Reply:
Why even keep them global? Why not a single temp buffer on the stack inside initialize_java_lang_classes?